### DISCUSSION TITLE
IOS version is lacking functionality in comparison with Android

### DISCUSSION THREAD

**Discussion by nl112358** [osmandapp/OsmAnd#11901] [General]
I recently changed phones, from Android to iPhone. Regrettably, the purchased features are not transferrable between the 2 eco systems. So I repurchased unlimited maps, Wikipedia en contour lines and hope that the development team will use it to add missing functionality to iOS:
- clickable tracks and ability to preview its height profile (distance or time on x-axis, height on y-axis)
- customizable icons for favourites (all my icons are now transformed in a simple star

Keep up the good work, really like your app!

**Comment by vshcherb** [comment]
Thanks for feedback, we hope to have it all (or the most needed) by the end of the year.
